handlers/bootstrap_artifact.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/mendersoftware/mender-artifact/artifact"
+)
+
+type BootstrapArtifact struct {
+	version    int
+	files      [](*DataFile)
+	typeInfoV3 *artifact.TypeInfoV3
+}
+
+func NewBootstrapArtifact() *BootstrapArtifact {
+	return &BootstrapArtifact{
+		version: 3,
+	}
+}
+
+func (b *BootstrapArtifact) GetVersion() int {
+	return b.version
+}
+
+// Return type of this update, which could be augmented.
+func (b *BootstrapArtifact) GetUpdateType() *string {
+	return nil
+}
+
+// Return type of original (non-augmented) update, if any.
+func (b *BootstrapArtifact) GetUpdateOriginalType() string {
+	return ""
+}
+
+func (b *BootstrapArtifact) GetUpdateDepends() (artifact.TypeInfoDepends, error) {
+	if b.typeInfoV3 == nil {
+		return nil, nil
+	}
+	return b.typeInfoV3.ArtifactDepends, nil
```

Comment:
Same here, unconditionally return the bottom result?

Reply:
done!